src/frontend/src/lib/components/hero/Actions.svelte

```diff
@@ -31,44 +32,46 @@
 	$: convertBtc = $tokenCkBtcLedger && $erc20UserTokensInitialized;
 </script>
 
-<div role="toolbar" class="flex w-full gap-6 justify-center pt-10 pb-3 px-1 max-w-96">
-	{#if $networkICP}
-		<IcReceive token={$tokenWithFallback} />
-	{:else if $networkEthereum}
-		<EthReceive />
-	{:else if $pseudoNetworkChainFusion}
-		<Receive />
-	{/if}
-
-	{#if $networkICP}
-		<IcSend token={$tokenWithFallback} />
-	{:else if $networkEthereum}
-		<EthSend token={$tokenWithFallback} />
-	{:else if $pseudoNetworkChainFusion}
-		<Send />
-	{/if}
-
-	{#if convertEth}
+<div role="toolbar" class="flex w-full justify-center pt-10 pb-3">
+	<HeroButtonGroup>
 		{#if $networkICP}
-			<ConvertToEthereum />
-		{:else}
-			<ConvertToCkETH />
+			<IcReceive token={$tokenWithFallback} />
```

Comment:
No logical changes right? The GitHub view add an empty block so just want to be sure

Reply:
yes, correct! I am not really sure why Github shows it like that, but the only change is that I wrap everything in `HeroButtonGroup`. My IDE view:

<img width="1139" alt="Screenshot 2024-09-20 at 13 55 20" src="https://github.com/user-attachments/assets/1739431a-2678-46c7-a994-13dc5ffc316f">
<img width="1143" alt="Screenshot 2024-09-20 at 13 55 30" src="https://github.com/user-attachments/assets/54cfce12-5b2b-44b5-a9de-f1e50d2feaed">